src/include/gurt/dlog.h

```diff
@@ -113,10 +114,13 @@ enum d_log_flag_bits {
 	D_LOG_SET_AS_DEFAULT	= 1U,
 };
 
-#define DLOG_PRISHIFT   24		/**< to get non-debug level */
-#define DLOG_DPRISHIFT  8		/**< to get debug level */
-#define DLOG_FACMASK    0x000000ff	/**< facility mask */
-#define DLOG_UNINIT	0x80000000	/**< Reserve one bit mask cache */
+#define DLOG_PRISHIFT     24		/**< to get non-debug level */
+#define DLOG_DPRISHIFT    8		/**< to get debug level */
+#define DLOG_PRINDMASK    0x0f000000	/**< mask for non-debug level bits */
```

Comment:
Am I correct that the only purpose of this mask is to ensure we don't spit out D_EMIT messages when the user has requested messages to go to STDOUT/STDERR.   Is this to avoid logging these messages on client where default is to log to stdout ?

Reply:
That is one usage.  The macro DLOG_PRI, which uses DLOG_PRINDMASK, gives an index into the upper priority setting for a facility.  It is the index into the "norm" structure that contains the character string of the priority (DEBUG, INFO, ...);  Used in clog_pristr.  Originally, the code used a hardcoded mask of 7 after the shift was performed.  This brings the calculation of the index into the include file and it removes the hardcoded value in the function.